api/integreatly/v1alpha1/grafana_types.go

```diff
@@ -484,12 +484,21 @@ type GrafanaConfigAuthProxy struct {
 }
 
 type GrafanaConfigDataProxy struct {
+	DialTimeout                  *int `json:"dialTimeout,omitempty" ini:"dialTimeout,omitempty"`
```

Comment:
I'm unable to find the dialTimeout definition in the docs, this could be due to I'm reading the latest docs.
It also seems to not follow the same standard of doing _
This is where I looked: https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#dataproxy
Can you please share where you found it?